### DISCUSSION TITLE
How to send a heartbeat?

### DISCUSSION THREAD

**Discussion by 389103326** [mqttjs/MQTT.js#1931] [Q&A]
I use the default keepalive value(60), and set reconnectPeriod 2000，frequently receive offline event, I guess need to send heartbeat after the keepalive timeend. But I can't find a method to send a heartbeat, I find pingResp is a property.

**Comment by robertsLando** [comment]
open an issue and fill the required fields please. I think you may be using an old MQTTjs version
